docs/10_using_dandi.md

```diff
@@ -7,7 +7,7 @@ src="../img/dandi_structure.svg"
 alt="dandi_structure"
 style="width: 70%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
 
-The Clients are DANDI Python CLI and DANDI Web application and the Server side contains a RESTful API and DANDI JupyterHub. The file organization to store data together with metadata is called `Dandiset`.
+The Client site contains DANDI Python CLI and DANDI Web application and the Server side contains a RESTful API and DANDI JupyterHub. The file organization to store data together with metadata is called `Dandiset`.
 
```

Comment:
I think it should be the Client "side" (not "site") contains the DANDI Python CLI etc., so that it matches the figure that depicts the Client side and the Service side.